reference/price_lists.v3.yml

```diff
@@ -3518,31 +3518,25 @@ components:
     PriceRecordBase:
       type: object
       description: Common Price Record properties.
-      title: PriceRecord Base
       properties:
         price:
           type: number
           format: double
-          description: |
-            The list price for the variant mapped in a Price List. Overrides any existing or Catalog list price for the variant/product.
+          description: The list price for the variant mapped in a Price List. Overrides any existing or Catalog list price for the variant/product.
           x-required:
             - put
-          example: 3.99
```

Comment:
should this example be gone?

Reply:
The original PR didn't have them so I removed them. I would like to have examples in the docs, so if you think we should keep it, i'll re-add it.